lib/cloudregister/registerutils.py

```diff
@@ -2027,6 +2035,41 @@ def has_network_access_by_ip_address(server_ip):
     return True
 
 
+# ----------------------------------------------------------------------------
+def is_suma_instance():
+    """Return a list containing the SUMA product if present."""
+    products_list = glob.glob('/etc/products.d/*.prod')
+    suma_prod = [
+        product for product in products_list if 'suse' in product.lower() and
+        'manager' in product.lower()
```

Comment:
Let's hope that suse has no product that contains 'suse' and 'manager' in its name. Call me pedantic but couldn't we match on a clear name ? I'm not too deep into suma but I can imagine it has a unique identifier ?

Reply:
we can check the exact match which is `SUSE-Manager-Server.prod` as of today

Reply:
Updated the check for SUMA instance

Reply:
I like @rjschwei feedback regarding this matching

Reply:
my gut feeling is, I would just check for the exact .prod file to identify the product and that's it

Reply:
given the suse madness about names I might be wrong here, but in that case matching for 'suse' and 'manager' could also fail when they decide to name it suse-mgr or s.u.s.e.-mgr or maybe some other crazy name to make live of developers hard :-))

Reply:
the [code](href="https://confluence.suse.com/display/publiccloud/Roadmap">https:/) has changed ? 
can you see the new code, @schaefi ?